tests/integration/ambient/waypoint_test.go

```diff
@@ -788,6 +788,51 @@ spec:
 	})
 }
 
+func TestTCPRoute(t *testing.T) {
```

Comment:
Should we also enable the TCPRoute gateway api tests for ambient? Happy for that to be a fast-follow

Reply:
I am not sure there are any mesh TCPRoute tests upstream

Reply:
Ah right, my brain skipped a step and read waypoint == gateway. Disregard